cypress/integration/awards-for-all.js

```diff
@@ -0,0 +1,481 @@
+const faker = require('faker');
```

Comment:
It was getting to be a pain to constantly add an `.only` in these tests to run just the awards for all spec so I've split it out into a separate file which Cypress treats as a dedicated option in the UI.

I'd originally kept everything in a single file for performance reasons but in testing it looks like more recent versions of Cypress have improved the delay between suites so we can stand to break things out a little more.